.travis.yml

```diff
@@ -21,7 +21,7 @@ script:
     npm test -- --coverage
   - serve -s build &
   - npm run itest
-  - 'curl -H "Authorization: token fbc63650102b81eefdad27f363516cb105ea50b8" -X PUT -d '{"message": "Test upload1", "content": "'"$(base64 -w 0 kiali-browser-test.png)"'"}' "https://api.github.com/repos/kiali-bot/travis-screenshots/contents/kiali-ui/travis-build-${TRAVIS_BUILD_NUMBER}.png"'
+  - 'curl -s -H "Authorization: token fbc63650102b81eefdad27f363516cb105ea50b8" -X PUT -d {\"message\": \"Test upload1\", \"content\": \"$(base64 -w 0 kiali-browser-test.png)\"} "https://api.github.com/repos/kiali-bot/travis-screenshots/contents/kiali-ui/travis-build-${TRAVIS_BUILD_NUMBER}.png"'
```

Comment:
I think this is OK for testing, but once all is set, please delete the token, create a new one and encrypt it: https://docs.travis-ci.com/user/encryption-keys/

Reply:
@josejulio 
Unfortunately it's not possible to access encrypted envs in PR as noted in the link above.  There's a long discussion on travis-ci repo about this issue, and one reasonable workaround is to create a separate read-only Github account for this purpose and obfuscate the password in .travis.yml with base64.

> Please note that encrypted environment variables are not available for pull requests from forks.

Reply:
darn! right, i had the same issue and had to offload some of the work on a heroku server and be notified as a webhook.
Basically the server did read the travis log to comment a success/fail status (the password being in the server)

https://github.com/hawkular/hawkular-services/blob/master/.travis.yml#L22